<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TRIM3D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="objTW">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>191</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>obj Files</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="labelObj">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Objects</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEle">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Elements</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="eleTW">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>601</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Z</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>M(amu)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Density(/A3)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fraction</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Disp(eV)</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="ionTW">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>791</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Z</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>M(amu)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x(A)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>y(A)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>z(A)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>cosX</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>cosY</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>cosZ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>E(eV)</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="labelIon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addObjBT">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addEleBT">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addIonBT">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
